R/hypothesize.R

```diff
@@ -53,27 +91,34 @@ parse_params <- function(dots, x) {
   mu_ind <- grep("mu", names(dots))
   med_ind <- grep("med", names(dots))
   sig_ind <- grep("sigma", names(dots))
-
+  
   # error: cannot specify more than one of props, means, medians, or sds
   if ( length(p_ind) + length(mu_ind) + length(med_ind) + length(sig_ind) != 1 ){
-    stop("Parameter values should be only one of proportions, means, medians, or standard deviations.")
+    stop('Parameter values should be only one of `p`, `mu`, `med`, or `sigma`.')
```

Comment:
What about the hypothesis test that the slope in SLR is 0? I know we have the slope implemented in `calculate()` but maybe it never made it to `hypothesize`?

Reply:
@andrewpbray Good catch. I don't think we ever implemented that in `hypothesize()`. Do we want to test for values of the slope different from 0 as well?

Reply:
Yeah, I don't see why not.

Reply:
@andrewpbray OK. I'll try to add that in today too.